<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationWidget</class>
 <widget class="QWidget" name="ConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_common">
      <attribute name="title">
       <string>Allgemein</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_gui">
      <attribute name="title">
       <string>GUI</string>
      </attribute>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Frames per Second</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_interpreter">
      <attribute name="title">
       <string>Interpreter</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_console">
      <attribute name="title">
       <string>Konsole</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Verbose Levels</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_logging">
      <attribute name="title">
       <string>Logging</string>
      </attribute>
      <widget class="QCheckBox" name="log_check_activate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Dateilogging aktivieren</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="log_line_namebase">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="log_check_append">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>120</y>
         <width>181</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>an bestehende Datei anhängen</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="log_check_timestamp">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Timestamp im Dateinamen</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Namensbasis</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_serial">
      <attribute name="title">
       <string>Seriell</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>251</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Baudrate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="serial_combo_baudrate">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>9</number>
          </property>
          <item>
           <property name="text">
            <string>110</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>230400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>460800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>921600</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Datenbits</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Parität</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Stopbits</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Flusssteuerung</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="serial_combo_datenbits">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="serial_combo_paritaet">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>Gerade</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ungerade</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Keine</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Markierung</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Leerzeichen</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="serial_combo_stopbits">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="serial_combo_flusssteuerung">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Hardware</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Keine</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Xon / Xoff</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="serial_combo_comport">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>COM 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM 4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM 5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM 6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM 7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM 8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM 9</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>COM Port</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>50</y>
         <width>271</width>
         <height>126</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Read Interval Timeout</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="serial_line_RIT"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Read Total Timeout Multiplier</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="serial_line_RTTM"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Read Total Timeout Constant</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="serial_line_RTTC"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Write Total Timeout Multiplier</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="serial_line_WTTM"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Write Total Timeout Constant</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="serial_line_WTTC"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="button_accept">
       <property name="text">
        <string>Übernehmen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Verwerfen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
